tests/Dfc.CourseDirectory.WebV2.Tests/TestData/TestData.CreateProvider.cs

```diff
@@ -69,6 +69,34 @@ public partial class TestData
 
             return providerId;
         }
+
+        public async Task<Guid> CreateProvider(Provider providerToAdd)
```

Comment:
Use/adapt existing `TestData.CreateProvider` here (trying to keep the TestData method signatures separated from Cosmos models to ease transition away later)